frontend/src/composables/useNormXml.ts

```diff
@@ -37,12 +37,14 @@ export function useNormXml(
 
   // We want to also update the data with the data returned from the PUT-request.
   const data = ref<string | null>(null)
-  const getNormXml: UseFetchReturn<string> = useGetNormXml(eli)
+  const getNormXml: UseFetchReturn<string> = useGetNormXml(eli, undefined, {
+    refetch: true,
+  })
   watch(getNormXml.data, () => {
     data.value = getNormXml.data.value
   })
 
-  const putNormXml = usePutNormXml(newXml, eli)
+  const putNormXml = usePutNormXml(newXml, eli, undefined, { refetch: true })
```

Comment:
i'm not sure why refetch is set to true here, but i haven't really thought about it